ivy_tests/test_ivy/test_frontends/test_jax/test_jax_devicearray.py

```diff
@@ -260,6 +260,28 @@ def test_jax_special_rpow(
     )
 
 
+# __all__
+@handle_frontend_test(
+    fn_tree="jax.lax.add",  # dummy fn_tree
```

Comment:
this test should be for the function you're implementing instead of `jax.lax.add`